src/packages/next/pages/.well-known/change-password.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ *  This file is part of CoCalc: Copyright © 2022 Sagemath, Inc.
+ *  License: AGPLv3 s.t. "Commons Clause" – see LICENSE.md for details
+ */
+
+import { useRouter } from "next/router";
+import React, { CSSProperties, useEffect } from "react";
+
+export default function WellKnownPassword() {
```

Comment:
Did you forget to include part of the commit or a file, since this isn't used anywhere?

Reply:
I thought this is discovered automatically? I don't know where to include this.

Reply:
that's embarrassing for me -- I completely forgot how nextjs works!  You're of course completely right